src/order/closure.lean

```diff
@@ -118,6 +118,28 @@ c.closure_top
 lemma closure_le_closed_iff_le {x y : α} (hy : c.closed y) : x ≤ y ↔ c x ≤ y :=
 by rw [← c.closure_eq_self_of_mem_closed hy, le_closure_iff]
 
+lemma closure_closure_union_left {α : Type u} [semilattice_sup α] (c : closure_operator α)
```

Comment:
```suggestion
lemma closure_closure_sup_left {α : Type u} [semilattice_sup α] (c : closure_operator α)
```
Same below

Reply:
Should I thus rename `closure_union_closure_le` and other lemmas already in this file too?

Reply:
Yeah, I think so - if the lemma doesn't use the union symbol it shouldn't be named after it.

@b-mehta, I think you chose these names originally - does this seem fine to you?

Reply:
I've now renamed everything.

Reply:
> Yeah, I think so - if the lemma doesn't use the union symbol it shouldn't be named after it.
> 
> @b-mehta, I think you chose these names originally - does this seem fine to you?

Yup, I absolutely should have suggested `sup` rather than `union`.